hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -299,6 +299,7 @@ def various_numpy_annotations(
     ],
     ids=lambda x: x[0],
 )
+@pytest.mark.skipif(sys.version_info >= (3, 13), reason="changed name/alias")
```

Comment:
ah, I was thinking we'd just skip the two affected cases - `pytest.param()` lets you apply markers to individual elements of a parametrize, as above - but let's merge now and turn the test back on later.